views/Navbar.tsx

```diff
@@ -1,159 +1,260 @@
-import React, { useState, useEffect } from 'react';
-import { useSession } from 'next-auth/react'
-import type { Session } from 'next-auth'
-import Link from 'next/link';
-import { AiOutlineMenu, AiOutlineClose } from 'react-icons/ai';
+import React, { useState, useEffect } from "react";
+import { useSession } from "next-auth/react";
+import type { Session } from "next-auth";
+import Link from "next/link";
+import { AiOutlineMenu, AiOutlineClose } from "react-icons/ai";
 import LoginLogout from "../components/LoginLogout";
-import chromeLogo from '../public/chrome-logo.png'
-import Image from 'next/image';
-import RudderContext from '../components/RudderContext';
-import { getAndSetAnonymousIdFromLocalStorage } from '../utils/rudderstack_initialize';
-import { getAuthUserId, getAuthUserName } from '../utils/auth';
-import AppBar from '../components/AppBar';
-import VibinexDarkLogo from '../public/vibinex-dark-logo.png';
-import Button from '../components/Button';
+import chromeLogo from "../public/chrome-logo.png";
+import Image from "next/image";
+import RudderContext from "../components/RudderContext";
+import { getAndSetAnonymousIdFromLocalStorage } from "../utils/rudderstack_initialize";
+import { getAuthUserId, getAuthUserName } from "../utils/auth";
+import AppBar from "../components/AppBar";
+import VibinexDarkLogo from "../public/vibinex-dark-logo.png";
+import Button from "../components/Button";
 
 const Navbar = (props: { transparent: boolean }) => {
-	const chromeExtensionLink = "https://chrome.google.com/webstore/detail/vibinex/jafgelpkkkopeaefadkdjcmnicgpcncc";
-	const { rudderEventMethods } = React.useContext(RudderContext);
-	const session: Session | null = useSession().data;
+  const chromeExtensionLink =
+    "https://chrome.google.com/webstore/detail/vibinex/jafgelpkkkopeaefadkdjcmnicgpcncc";
+  const { rudderEventMethods } = React.useContext(RudderContext);
+  const session: Session | null = useSession().data;
 
-	const [showNavbar, setShowNavbar] = useState(false);
-	const [scrollDown, setScrollDown] = useState(!props.transparent);
+  const [showNavbar, setShowNavbar] = useState(false);
+  const [scrollDown, setScrollDown] = useState(!props.transparent);
```

Comment:
If you haven't followed this in your other files, I just want to emphasize that we use tabs (not spaces) in our project. If you have used spaces for indentation, please replace them with tabs.